src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -93,6 +94,22 @@ object SnowflakeTelemetry {
     }
   }
 
+  private[snowflake] def getSparkDependencies: Seq[String] = {
+    try {
+      val activeSparkSession = SparkSession.getActiveSession
+      if (activeSparkSession.nonEmpty) {
+        val sparkContext = activeSparkSession.get.sparkContext
+        (sparkContext.files ++ sparkContext.archives).distinct
```

Comment:
Is `sparkContext.archives` only going to pick up Python libs? And can you write a comment here about this?

Reply:
According to the document. It may be used for other purpose. https://spark.apache.org/docs/latest/configuration.html
``` 
Comma-separated list of archives to be extracted into the working directory of each executor. .jar, 
.tar.gz, .tgz and .zip are supported. You can specify the directory name to unpack via adding # after
 the file name to unpack, for example, file.zip#directory. This configuration is experimental.
```

Reply:
So we may have some duplication in `dependencies` with `libraries`, right?

Reply:
logically, it is possible. But I think archive/file are mainly for non-jar files.
We can also retrieve jar files by `SparkContext.jars`, this may be duplicated to `libraries`.

Reply:
```scala
SparkSession.getActiveSession
  .map(_.sparkContext)
  .map(context => (context.files ++ context.archives).distinct)
  .getOrElse(Seq.empty)
```